app/controllers/concerns/saml_idp_auth_concern.rb

```diff
@@ -148,9 +148,8 @@ def identity_needs_verification?
 
   def ial_context
     @ial_context ||= IalContext.new(
-      ial: requested_ial_authn_context,
+      ial: resolved_authn_context_int_ial,
```

Comment:
WDYT of renaming the attribute in `IalContext` to be `int_ial` for clarity?

Reply:
The `IalContext` actually takes an integer or a string.

Reply:
got it!